noir_stdlib/src/field.nr

```diff
@@ -53,3 +53,24 @@ fn modulus_be_bytes() -> [u8] {}
 
 #[builtin(modulus_le_bytes)]
 fn modulus_le_bytes() -> [u8] {}
+
+// Returns true if x is less than y
+// TODO: consider sftey of final few bits
+fn lt_bytes32(x: Field, y: Field) -> bool {
```

Comment:
We do not allow comparison of field elements so this function must be removed from this file.

Reply:
One of the required conditions for checking the validity of an EdDSA signature is that the $s$ parameter should be less than the subgroup order of the curve over which the signature is defined (see [RFC8032 5.1.7](https://www.rfc-editor.org/rfc/rfc8032#page-14)). 
How do I check that if field element comparisons aren't allowed?

Reply:
I moved it to `eddsa.nr`
Btw, what's the rationale behind not allowing field element comparisons in `field.nr`?

Reply:
Thank you :)
Don't ask me this, I was pushing for having it!

Reply:
Explanation for posterity: https://the.dev/posts/cryptography/circuit_design/integers_in_circ/
tldr: Finite fields don't have any notion of ordering

Reply:
Finite fields do have a notion of ordering as the simple 'less than' defines an ordering on the set.
However, is a 'set ordering', not a 'field ordering'.